src/main/resources/assets/tropicraft/models/block/bamboo_item_frame_map.json

```diff
@@ -0,0 +1,56 @@
+{
+    "textures": {
+        "particle": "tropicraft:blocks/bamboo_Side",
+        "wood": "tropicraft:blocks/bamboo_Side",
```

Comment:
lol, we left the texture ID as wood. If I was picky I would say change it, but it's functional and doesn't make a big difference so your call on this one.